detectors/node/opentelemetry-resource-detector-container/test/ContainerDetector.test.ts

```diff
@@ -137,4 +138,146 @@ describe('ContainerDetector', () => {
       assertEmptyResource(resource);
     });
   });
+
+  describe('Detect containerId in k8 environment', () => {
+    it('should return an empty Resource when not in a K8 environment', async () => {
```

Comment:
Could please change all uses of "k8" to "k8s" (upper and lower case). AFAIK "k8s" is the more common abbreviation -- at least "k8s" is used as a prefix in some OpenTelemetry specs / semantic conventions.

Reply:
Absolutely